testutils/resourcebuilder/machine/v1beta1/machine.go

```diff
@@ -140,6 +145,12 @@ func (m MachineBuilder) WithProviderSpecBuilder(builder resourcebuilder.RawExten
 	return m
 }
 
+// WithProviderID sets the providerID field for the machine builder.
+func (m MachineBuilder) WithProviderID(providerID string) MachineBuilder {
```

Comment:
Nit, ProviderID is alphabetically ahead of ProviderSpec. I believe everything else here is alphabetically sorted so nice to keep that

Reply:
Ah yeah sure :) updated.